webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -132,6 +129,20 @@ export function setErrorMessage(
   }
 }
 
+export function replaceStream(oldStreamId: String, newStream: MediaStream) {
```

Comment:
```suggestion
export function replaceStream(oldStreamId: String, newStream: MediaStream): void {
```

Reply:
We didn't add `: void` in other functions' definition. Do we want to put it everywhere?

Reply:
I guess the answer is yes :)

Reply:
Go resolve if done